packages/styles/scss/patterns/blocks/callout-quote/index.scss

```diff
@@ -8,3 +8,20 @@
 @import '../../../globals/imports';
 @import '../../../internal/callout/callout';
 @import '../../../components/quote/quote';
+@import '../../../globals/utils/hang';
+
+@mixin callout-quote {
+  .#{$prefix}--callout-quote {
+    @include carbon--breakpoint('lg') {
```

Comment:
Since you're declaring the `hang` mixin at the md breakpoint, I don't think we need the `lg` breakpoint too